version: "3.7"

services:
  cloudiolab:
    image: docker.cloudiolab.com/cloudiolab/cloudiolab:1.0
    ports:
      - "3001:3001"
    networks:
      - cloudiolab_common_network
    deploy:
      replicas: 1
      update_config:
        failure_action: rollback
        parallelism: 2
        delay: 10s
      restart_policy:
        condition: on-failure
      placement:
        constraints: [node.hostname == jarvis]

networks:
  cloudiolab_common_network:
    external: true
